src/components/ErrorMessage.js

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+
+const ErrorMessage = ({ error }) => (
+  <div className="search__error">
+    <h2>{error ? 'Something went wrong' : 'Not found'}</h2>
```

Comment:
I love the idea of surfacing an error when there’s, e.g., no network, but if there are simply no pins we could probably skip showing an error. What do you think?

Reply:
At the moment, we’re not showing errors if there’re no pins to render.
What gets displayed is a  map 
![no pins](https://user-images.githubusercontent.com/16920088/126694590-4381f5a7-3496-4d70-a0e3-e6b2292f00ce.png)